scripts/find-updates.sh

```diff
@@ -0,0 +1,138 @@
+current_version=$(grep appVersion assets/components/cert-manager/manifests/Chart.yaml | cut -d":" -f2)
+
+# Download the latest chart in temp dir and find out the version in it.
+tmpdir=$(mktemp -d)
+cd "${tmpdir}"
+helm repo add jetstack https://charts.jetstack.io >/dev/null 2>&1
+helm repo update >/dev/null 2>&1
+helm fetch --untar --untardir ./ jetstack/cert-manager
+version=$(grep appVersion "${tmpdir}/cert-manager/Chart.yaml" | cut -d":" -f2)
+
+printf "${format}" "cert-manager" "${current_version}" "${version}"
+
+###########################
+# contour
+
+cd "${workdir}"
+current_version=$(grep "image: docker.io/projectcontour/contour" assets/components/contour/contour/03-contour.yaml | cut -d":" -f3)
+
+get_latest_release projectcontour/contour
+printf "${format}" "contour" "${current_version}" "${version}"
+
+###########################
+# Dex
+cd "${workdir}"
+current_version=$(grep "image: quay.io/dexidp/dex" pkg/components/dex/component.go | cut -d":" -f3)
+
+get_latest_release dexidp/dex
+printf "${format}" "Dex" "${current_version}" "${version}"
+
+
+###########################
+# external dns
+cd "${workdir}"
+current_version=$(grep version assets/components/external-dns/manifests/Chart.yaml | cut -d":" -f2)
+
+tmpdir=$(mktemp -d)
+cd "${tmpdir}"
+helm repo add bitnami https://charts.bitnami.com/bitnami >/dev/null 2>&1
+helm repo update >/dev/null 2>&1
+helm fetch --untar --untardir ./ bitnami/external-dns
+version=$(grep version "${tmpdir}/external-dns/Chart.yaml" | cut -d":" -f2)
+
+printf "${format}" "external-dns" "${current_version}" "${version}"
+
+###########################
+# gangway
+
+cd "${workdir}"
+current_version=$(grep "image: gcr.io/heptio-images/gangway" pkg/components/gangway/component.go | cut -d":" -f3)
+
+get_latest_release heptiolabs/gangway
+printf "${format}" "gangway" "${current_version}" "${version}"
+
+###########################
+# metallb
+
+cd "${workdir}"
+current_version=$(grep "image: quay.io/kinvolk/metallb-controller" pkg/components/metallb/manifests.go | cut -d":" -f3)
+
+get_latest_release metallb/metallb
+printf "${format}" "metallb" "${current_version}" "${version}"
+
+###########################
+# metrics-server
+cd "${workdir}"
+current_version=$(grep version assets/components/metrics-server/Chart.yaml | cut -d":" -f2)
+
+tmpdir=$(mktemp -d)
+cd "${tmpdir}"
+
+helm repo add stable https://kubernetes-charts.storage.googleapis.com >/dev/null 2>&1
+helm repo update >/dev/null 2>&1
+helm fetch --untar --untardir ./ stable/metrics-server
+version=$(grep version "${tmpdir}/metrics-server/Chart.yaml" | cut -d":" -f2)
+
+printf "${format}" "metrics-server" "${current_version}" "${version}"
+
+###########################
+# openebs
+cd "${workdir}"
+current_version=$(grep version assets/components/openebs/Chart.yaml | cut -d":" -f2)
+
+tmpdir=$(mktemp -d)
+cd "${tmpdir}"
+
+helm fetch --untar --untardir ./ stable/openebs
+version=$(grep version "${tmpdir}/openebs/Chart.yaml" | cut -d":" -f2)
```

Comment:
```suggestion
version=$(grep version "${tmpdir}/openebs/Chart.yaml" | cut -d":" -f2 | sed 's/ //g')
```

Reply:
Actually, following components produce extra lines, so this fix should be applied everywhere:
- cert-manager
- dex
- external-dns
- metrics-server
- openebs

Reply:
But I don't see new lines :thinking: 

```console
$ ./scripts/find-updates.sh 
Component               Current Version     Latest Version
---------               ---------------     --------------
kubernetes                      v1.18.3           v1.16.10
calico                          v3.14.1            v3.14.1
etcd                             v3.4.7             v3.4.9
cert-manager                    v0.14.2            v0.15.1
contour                          v1.3.0             v1.5.0
dex                             v2.23.0            v2.24.0
external-dns                     2.21.2              3.1.0
gangway                          v3.2.0             v3.2.0
metallb              v0.8.3-2-gf653773b             v0.9.3
metrics-server                   2.11.1             2.11.1
openebs                           1.9.0             1.10.2
```

Reply:
Fixed the kubernetes problem(it showed 1.16.x):

```console
$ ./scripts/find-updates.sh 
Component               Current Version     Latest Version
---------               ---------------     --------------
kubernetes                      v1.18.3            v1.18.3
calico                          v3.14.1            v3.14.1
...
```

Changed this line: https://github.com/kinvolk/lokomotive/pull/375/files#diff-77dda93dfb64019995d755375867dd5bR29

Reply:
It seems `printf` formatting fixes this. But the value is still not entirely correct. See:
```
$ echo "Version: '$(grep appVersion assets/components/cert-manager/manifests/Chart.yaml | cut -d":" -f2)'"
Version: ' v0.14.2'
```